src/current/v24.3/troubleshoot-replication-zones.md

```diff
@@ -0,0 +1,219 @@
+---
+title: Troubleshoot Replication Zones
+summary: Troubleshooting guide for replication zones, which control the number and location of replicas for specific sets of data.
+keywords: ttl, time to live, availability zone
+toc: true
+docs_area: manage
+---
+
+This page has instructions showing how to troubleshoot scenarios where you believe replicas are not behaving as specified by your [zone configurations]({% link {{ page.version.version }}/configure-replication-zones.md %}).
+
+{{site.data.alerts.callout_danger}}
+{% include {{ page.version.version }}/zone-configs/avoid-manual-zone-configs.md %}
+{{site.data.alerts.end}}
+
+## Prerequisites
+
+This page assumes you have read and understood the following:
+
+- [Replication controls > Replication zone levels]({% link {{ page.version.version }}/configure-replication-zones.md %}#replication-zone-levels), which describes how the inheritance hierarchy of replication zones works. **This is critical to understand for troubleshooting.**
+- [Monitoring and alerting > Critical nodes endpoint]({% link {{ page.version.version }}/monitoring-and-alerting.md %}#critical-nodes-endpoint), which is used to monitor if any of your cluster's ranges are under-replicated, or if your data placement constraints are being violated.
+- [`SHOW ZONE CONFIGURATIONS`]({% link {{ page.version.version }}/show-zone-configurations.md %}), which is the SQL statement used to view details about the replication zone configuration of various schema objects.
+
+## Types of problems
+
+The most common types of problems you may encounter when [manually configuring replication zones]({% link {{ page.version.version }}/configure-replication-zones.md %}#manage-replication-zones) are:
+
+1. The _replica location_ problem, a.k.a., "The replicas are not _where_ they should be". For replica location problems, the following zone config variables may be involved:
```

Comment:
```suggestion
1. The _replica location_ problem, a.k.a., "The replicas are not **where** they should be". For replica location problems, the following zone config variables may be involved:
```

Reply:
Also recommend linking "zone config variables" to configure-replication-zones#replication-zone-variables

Reply:
done and done!